DRODLib/GameConstants.cpp

```diff
@@ -93,7 +93,7 @@ namespace InputCommands
 		keyDefinitions[DCMD_PuzzleModeOptions] = new KeyDefinition(CMD_EXTRA_PUZZLE_MODE_OPTIONS, "Key_PuzzleModeOptions", MID_Command_PuzzleModeOptions, BuildInputKey(SDLK_F3, false, true, false));
 		keyDefinitions[DCMD_ToggleTurnCount] = new KeyDefinition(CMD_EXTRA_TOGGLE_TURN_COUNT, "Key_ToggleTurnCount", MID_Command_ToggleTurnCount, SDLK_F7);
 		keyDefinitions[DCMD_QuickDemoRecord] = new KeyDefinition(CMD_EXTRA_QUICK_DEMO_RECORD, "Key_QuickDemoRecord", MID_Command_QuickDemoRecord, SDLK_F4);
-		keyDefinitions[DCMD_ToggleDemoRecord] = new KeyDefinition(CMD_EXTRA_TOGGLE_DEMO_RECORD, "Key_ToggleDemoRecord", MID_Command_ToggleDemoRecord, SDLK_F5);
+		keyDefinitions[DCMD_ToggleDemoRecord] = new KeyDefinition(CMD_EXTRA_TOGGLE_DEMO_RECORD, "Key_ToggleDemoRecord", MID_Command_ToggleDemoRecord, BuildInputKey(SDLK_F6, false, false, true));
```

Comment:
Why are we shifting from F5 to F6?

Reply:
It's a side effect of allowing these keys to be rebinded. Due to how input -> command lookup works, if multiple commands share a key, only one will be recognised. The player can't put themselves in that situation, but some of the default keys collide.

There is probably a clever way to allow non-clashing commands to share keys, but this is the simple fix.

Reply:
Thank you. This makes sense.

I'd expect we need to update the help doc that mentions this key/case. Would you please check and make that change?